deps/rabbit/src/rabbit_quorum_queue.erl

```diff
@@ -1008,46 +1010,58 @@ consume(Q, Spec, QState0) when ?amqqueue_is_quorum(Q) ->
                      args => Args,
                      username => ActingUser,
                      priority => Priority},
-    {ok, _Infos, QState} = rabbit_fifo_client:checkout(ConsumerTag,
-                                                       Mode, ConsumerMeta,
-                                                       QState0),
-    case single_active_consumer_on(Q) of
-        true ->
-            %% get the leader from state
-            case rabbit_fifo_client:query_single_active_consumer(QState) of
-                {ok, SacResult} ->
-                    ActivityStatus = case SacResult of
-                                         {value, {ConsumerTag, ChPid}} ->
-                                             single_active;
-                                         _ ->
-                                             waiting
-                                     end,
+    case rabbit_fifo_client:checkout(
```

Comment:
Try to favour left to right formatting styles as used in this module. so arguments should as far as possible go to the right of the function name.